### PR TITLE
improve theme buton

### LINKED ISSUE #159: Improve theme button location
Theme button location is not natural I think, I already expressed it in https://github.com/edamontology/edam-browser/pull/150#issuecomment-828484969 I even removed the button from the edit page with 095b1ec8ad54d3c3147459b8972f6d47275773cc. I think work need to be done to find a proper location. UX/UI skill might me needed.

### PR DESCRIPTION
**Pull request by sunny775**
This PR improves theme button in terms of position and appearance

- [x] I indicated which issue (if any) is closed with this PR using [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] I only changed lines related to my PR (no bulk reformating)
- [x] I indicated the source and check the license if I re-use code, or I did not re-use code
- [x] I made my best to solve only one issue in this PR, or explain why multi had to be solved at once.

### CODE REVIEW

**Comment by raashika03** [comment]
Screenshot from 2021-04-30 13-35-34
Hi @sunny775 would you like to check this?
Some space is left there in the right side for smaller screens.
I tried your code locally.

File: index.html [hunk 1]
```diff
@@ -193,11 +202,19 @@ <h1 id="pageTitle">EDAM ontology</h1>
                     </div>
                 </div>
             </div>
-            <button class="btn-toggle" title="Change theme mode">
-            <svg style="fill: var(--color-profile-color-modes-toggle-moon);" aria-hidden="true" width="20" height="20" viewBox="0 0 14 13" xmlns="http://www.w3.org/2000/svg">
-                <path fill-rule="evenodd" clip-rule="evenodd" d="M4.52208 7.71754C7.5782 7.71754 10.0557 5.24006 10.0557 2.18394C10.0557 1.93498 10.0392 1.68986 10.0074 1.44961C9.95801 1.07727 10.3495 0.771159 10.6474 0.99992C12.1153 2.12716 13.0615 3.89999 13.0615 5.89383C13.0615 9.29958 10.3006 12.0605 6.89485 12.0605C3.95334 12.0605 1.49286 10.001 0.876728 7.24527C0.794841 6.87902 1.23668 6.65289 1.55321 6.85451C2.41106 7.40095 3.4296 7.71754 4.52208 7.71754Z"></path>
-            </svg>
-            </button>
+            
+            <div class="pull-right">
```
Comment (bryan-brancotte) [inline index.html:206]: Why re-adding recently removed code ? I think it is a side effect of the force-push you use, as a general habit, do no use it if you don't have too. I think here the merge failed but as you force push thing we cannot see it in the git history.

File: css/dark-theme.css [hunk 2]
```diff
@@ -74,6 +74,22 @@
   color: #B8B8B8;
   border-color: #050505;
 }
+div.btn-toggle-wrapper {
+  position: absolute;
+  top: 12px;
+  right: -160px;
```
Comment (bryan-brancotte) [inline css/dark-theme.css:80]: you had to set it to -160px as a consequence of locating the button inside the #page-header in a DOM-tree point of view, and thus colliding with its layout. As you propose to use position absolute, I would recommend that you put it out of the header (in a DOM-tree pov).

**Review by bryan-brancotte — COMMENTED** [review]
Thanks for your proposition, also keep in mind that when doing PR with visual impact, a screenshot is always welcome (I'm adding it to the PR template)

File: index.html (hunk 1 above)
Comment (sunny775) [reply index.html:206]: You're right. There was a merge conflict. I think it was in trying to resolve the conflict that I erroneously re-introduced the recently removed code

File: css/dark-theme.css (hunk 2 above)
Comment (sunny775) [reply css/dark-theme.css:80]: Ok. I'll do that and commit the changes asap

Comment (sunny775) [reply css/dark-theme.css]: I've moved it out of the header

File: index.html (hunk 1 above)
Comment (sunny775) [reply index.html]: I've resolved it

**Comment by sunny775** [comment]
@bryan-brancotte  here are the images of the new look:

new_btn_position

dark_m_btn_position

**Review by bryan-brancotte — APPROVED** [review]
Hi @sunny775 

Good job :clap: :+1: 

merging it !

**Comment by sunny775** [comment]
Thanks @bryan-brancotte

**Pull request merged**